src/openzaak/components/zaken/models/zaken.py

```diff
@@ -699,13 +718,23 @@ class ZaakInformatieObject(models.Model):
     class Meta:
         verbose_name = "zaakinformatieobject"
         verbose_name_plural = "zaakinformatieobjecten"
-        unique_together = ("zaak", "informatieobject")
+        unique_together = (
+            ("zaak", "_informatieobject"),
+            # ("zaak", "_informatieobject_url"),
```

Comment:
Should be uncommented

Reply:
Uncomment would be a bit to strict (since the URLField is not allowed to be nullable -> empty strings will trip it up).

Added a conditional `UniqueConstraint`.